### ISSUE TITLE
Discussion: An optimal coding schema for our research purposes

### ISSUE THREAD

**Issue by ragsol** [jilltxt/machinevision#57]
I have cross-referenced the GitHub issues with the Database and Jill's project proposal, and realized that there are several questions we're not really answering at the moment in our tags (and several that we probably don't need to answer that we do). I know projects change over time, but this is an attempt to customize the schema (schema/scheme?) to better fit what we set out to do (and what it seems to me we need for our future research). The suggestions below are by no means the only answers, and I want us to discuss these points together because, as we have learned before, we think better together :) Names and ideas are all up for discussion. I do hope that we can use this to start a conversation about what can be done to remedy some of the issues in our schema - before we reach 1000..! 

[PDF suggestion for database](https://github.com/jilltxt/machinevision/files/3550623/Database.proposal.2.pdf)

_(you probably need to zoom a lot, because my XMind is only a free version :) The yellow bits are the ones discussed below)_

#### Publication type
- "Comic"
Add it, because we already need it.
- "Advertisment"
I don't know if we have any ads registered at the moment, but it seems to me that even though they're "everyday life", they're a bit on the side of all the other categories (and they open a can of worms of what we "should" log). I suggest removing this category.

#### Technology used (and referenced)
- "Analogue"
Again, doesn't this category open up for a lot of tags outside of our main focus area? Perhaps it is tagged with several works, but even if the analogue camera changes perception etc. it's not really matching the other (digital) technologies on the list.
- "Camera"
This is a confusing one, and must be applied properly. I looked into it in order to see if it was necessary, and then I remembered I vouched for it with the camcorder games. However, those enable night vision, so now that we have that tag, my need for Camera isn't as pressing. There are several works tagged with this, and it seems to me that many of them are wrongly tagged, because the work "happens to have a camera there". This was meant to be a specific kind of camera. If we don't delete this category, we definitely need to revisit the works tagged with it.
- "Facial recognition"
So perhaps this is unecessary, but a way of structuring the list that will also make it easier in export is to make subcategories, just as with the genre. Facial recognition is also biometrics, so exporting biometrics will this way include FR, but you can still also separate them. 
- "Infrared"
Has been discussed earlier, was suggested for deletion (because it's now in Night vision)
- "Neural networks"
Has been listed as "delete" for a long time now, so make sure that all works are registered elsewhere, and then off you go :)

#### Visual data presentation
Okay, so here we venture into unknown waters. I believe we need something to tackle the issue of whether or not the visual data is presented as being an objective lens to the real world or a malleable or even outright fake picture within the fictional work. This is not what it actually is (because we know all digital presentations are mediated), but what the work presents. For instance, surveillance cameras usually present as objective lenses (you see exactly what's happening somewhere else), whereas deepfakes sort of gives it away in the name. For the distinction distorted/fake, I wanted to avoid a binary, but I also believe there's a difference between a glitchy image (distorted) and deepfake.

#### User position
This is another suggested new one. Jill's proposal talks a lot about how the player/reader/user/etc. is positioned in relation to the MV. We tried this before, but ended up letting the characters be the determiners. But what if we make it really easy: is the user experiencing something fixed or something they can act directly upon? View/Read will be applicable for films, a lot of literature, and several artworks. Interact/Choose will be games, but also E-lit, artworks, that crazy TV show with Bear Gryllz, etc.

#### Themes
A radical suggestion from me: deleting themes. It's a pain in the butt and I'm not sure what we're using it for. We could obviously keep it and I won't complain, but without it, the work will be "on the work's premise", and then the Situations are where we get to revel in our interpretations. Several comments we get is on the obvious subjectivity of the schema, which is both fine and unavoidable, but perhaps this is a way of "tidying". Another suggestion is that we change it to "Features", and then tag key items that the work features, such as cities, labor, nature, war. A lot of the words we have now seem more descriptive, and then we also have a sort of "visual representation" of the work within the database. Or we could just call Ragnhild crazy and leave it as is. (I haven't reviewed the different themes, so if we decide to leave it as is, I'll get right to it ;))

#### (work/overall) Sentiment
This is also a very problematic one that I would suggest deleting. We have sentiments in the situations, and this one (at least for me) is very often tagged with Mixed. I'll not deny that the same reasoning as for Themes is at play here, as well as minimizing the vast amount of tags we have to do in order to register a work... but! I do feel this category is more redundant now that we carefully tag this in the Situations.

#### Situational function
- "Hiding/Deleting"
Create, as suggested in GitHub (because it's needed, and because it's a logical opposite to Revealing).
- "Interpreting"
Consider deleting, as suggested in GitHub. Overlaps with Understanding and Analyzing.
- "Matching"
Consider deleting, as suggested in GitHub. Overlaps with Identifying, but the latter is more generalized and captures both. 
- "Organizing"
Create, as suggested in GitHub.
- "Recording"
Consider deleting. It is a different sentiment to this word than to Capturing, but it overlaps a lot. Recording is the more generalized of the two, and indicates some sort of long-term perspective, but we have this in the Societal function with Remembering.
- "Targeting"
Consider deleting. Overlaps with Locating, but the latter is more generalized and captures both. 
- "Understanding"
Consider deleting, as suggested in GitHub. Overlaps with several words, although it has a slightly different meaning. Does Analyzing cover it? I suggest keeping one of the three: Analyzing, Interpreting, Understanding.

#### Societal function
- "Decorating"
Change the name from Decorating/Embellishing
- "Illustrating"
Change the name from Informing/Illustrating/Demonstrating
- "Inspiring"
Change the name from Inspiring/Motivating. 
- "Provoking"
Change the name from Inciting/Provoking. 

#### Characteristics
- "Immigrant"
Suggest deleting, because it's encompassed by the new category Multiple/Complex.
- "Trans"
As mentioned in a different thread, we still need to add this term in Gender.
- "Other"
Suggest grouping the words that don't belong neatly in the other categories. 
- "Group"
Catering to those instances where there is more than one person, but not specified individuals. Now the tags are all over the place, but perhaps it can be easily fixed by a simple value/non-value tag?

#### Technological presence
A suggested new category to catch more nuance from the initial research proposal (and, incidentally, to give my research more valuable data). The idea is that how the operator of machine vision feels about the MV is important (as we have tagged before), but this allows for a simpler tagging by showing if they believe it to be a natural part of their everyday life or if it's new and strange. It also opens for merging with the now Aware category, and can be used on both Operator and Viewed. Suggested tags: Alien/Strange, Functional (this probably needs a better word), Habitualized, Unaware. Blank for no value or unknown.

#### Agency
This mimics "User position", but is specific to the Situation. I tried making the words for the two groups the same, but it doesn't translate... For this one: does the character/art aficionado Watch a surveillance stream, Control camera positioning, or put on filters or change an algorithm (Change/Manipulate/Distort)? One key question here is if this can be translated to the Viewed? 

#### Aware of being viewed
Delete: In this suggestion, this is implemented elsewhere.

#### Who owns/developed the technology
I suspect this category is floating around a bit. I recommend checking the tags and seeing if we have too many.

#### Relationship human/machine vision
Delete: In this suggestion, this is implemented elsewhere. Also, this is one of the most hated tags in the database 🤣

**Comment by ragsol** [comment]
And a final comment: Perhaps we shouldn't use one technology in the title of a Situation. It reduces the technologies actually present in a Situation, and dictates the tagging and the reading. Possible solutions (feel free to add more) : using letters/numbers, using abbreviations for all technologies in the Situation ++

**Comment by LinnHeidi** [comment]
I am sure we discussed this, and I will just add this here because is more of a thought than an issue.

We have situations for when MV is used, but what about when MV is more the backdrop? 
It is a bit opposite to what we discussed with Jill`s situation from the Atwood book. No MV theme in the book, but has one MV situation. 

What if a story has a narrative where MV is often referred to casually and not in depth? It`s important to log it still. But should we have an alternative to situation? Situation to me is so "active" something is happening, and doing. But maybe there could be an option when MV is more of a backdrop? casual and passive. I have experienced especially by reading, but also when watching movies that there are contexts that are very machine visiony but does not feel enough to make a situation out of it?  I think maybe this can connect with also the "interactive" bit?
